python/beeai_framework/backend/chat.py

```diff
@@ -42,7 +42,7 @@ class ChatModelParameters(BaseModel):
     max_tokens: int | None = None
     top_p: int | None = None
     frequency_penalty: int | None = None
-    temperature: int | None = None
+    temperature: int | None = 0
```

Comment:
if the default is 0 now should None still be an option?
```suggestion
    temperature: int = 0
```